app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1972,6 +1965,17 @@ public boolean onCreateOptionsMenu(Menu menu) {
         return false;
     }
 
+    @Override
+    public boolean onPrepareOptionsMenu(Menu menu) {
+        if (networkStatus.equals(Constants.STATUS_CONNECTED)) {
+            menu.findItem(R.id.refresh).setEnabled(true);
```

Comment:
Device is connected to server but **Refresh** option is not enabled. 
![ezgif com-video-to-gif 7](https://cloud.githubusercontent.com/assets/18511177/23401731/ce4c8d8e-fdce-11e6-9756-dad300b72045.gif)